### DISCUSSION TITLE
OIDC authentication template

### DISCUSSION THREAD

**Discussion by Kyemate** [bitfoundation/bitplatform#7052] [Ideas]
OIDC authentication  is a pretty common use case  that is missing from the templates.

**Comment by yasmoradi** [comment]
Greetings @Kyemate 
Could you please clarify your requirements?
Regards

**Reply by Kyemate** [reply]
Hi,
 I was thinking that being able to login through external authentication providers with the bit boilerplate templates. For example openid is a pretty common use case today when a lot of apps don't want to handle their on auth and instead relay on google, facebook etc.

best regards

**Reply by yasmoradi** [reply]
It seems that aspnetcore identity provides such a feature, but I've not tested it yet!
